js/customize-snapshots-admin.js

```diff
@@ -0,0 +1,73 @@
+( function( $ ) {
+	'use strict';
+
+	$( function() {
+		var $link, linkText, linkActions, dataSlug, initializeLink;
+
+		$link = $( '.snapshot-toggle-setting-removal' );
+		linkText = [ 'Remove setting', 'Restore setting' ];
+		linkActions = [ 'remove', 'restore' ];
+		dataSlug = 'cs-action';
+
+		initializeLink = function() {
+			$link.text( linkText[ 0 ] )
+				.data( dataSlug, linkActions[ 0 ] );
+		};
```

Comment:
Thanks, [this commit](https://github.com/xwp/wp-customize-snapshots/pull/84/commits/35d5bca1f03a9fb5cd1aef66a355b91dd30eba33#diff-36a80da8dbba0bdbe840617270bb1182R363) outputs the translated strings with PHP.